srtcore/core.h

```diff
@@ -943,13 +943,41 @@ class CUDT
     ///
     /// @returns the nmber of packets sent.
     int  sendCtrlAck(CPacket& ctrlpkt, int size);
+    void sendLossReport(const std::vector< std::pair<int32_t, int32_t> >& losslist);
 
     void processCtrl(const CPacket& ctrlpkt);
-    void sendLossReport(const std::vector< std::pair<int32_t, int32_t> >& losslist);
-    void processCtrlAck(const CPacket& ctrlpkt, const time_point &currtime);
+    
+    /// @brief Process incoming control ACK packet.
+    /// @param ctrlpkt incoming ACK packet
+    /// @param currtime current clock time
+    void processCtrlAck(const CPacket& ctrlpkt, const time_point& currtime);
```

Comment:
Consider changing `currtime` to `tsArrival` to have the consistency between `processCtrlAck` and `processCtrlAckAck` plus changing the attribute description to `time of packet arrival`.

Reply:
`processCtrlAck` expects the current time, as it will be used to set the time of the last ACKACK sending (`m_SndLastAck2Time`). The accuracy is not that important in this case.
While `processCtrlAckAck` uses `tsArrivalTime` to calculate RTT, therefore it should be as close to packet arrival time as possible.
As of now, the same time is passed in both functions. Hopefully, later there would be two times available: current and packet arrival.